drivers/include/net/netdev.h

```diff
@@ -469,7 +469,19 @@ static inline int netdev_set_notsup(netdev_t *dev, netopt_t opt,
     return -ENOTSUP;
 }
 
-
+/**
+ * @brief Informs netdev there was an interrupt request from the network device.
+ *
+ *        This function calls @ref event_callback with NETDEV_EVENT_ISR event.
```

Comment:
The `@ref event_callback` is undefined here (see travis)

Reply:
```suggestion
 *        This function calls netdev_t::event_callback with NETDEV_EVENT_ISR event.
```